src/LaunchDarkly.EventSource/LaunchDarkly.EventSource.csproj

```diff
@@ -49,5 +49,6 @@
 
   <PropertyGroup>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\LaunchDarkly.EventSource.xml</DocumentationFile>
+    <AssemblyOriginatorKeyFile>../../LaunchDarkly.EventSource.snk</AssemblyOriginatorKeyFile>
```

Comment:
I don't know why this change was made, but please revert it. We do not want the assembly to be strong-named in _every_ configuration, but only in the Release configuration, as is specified on line 33 of this file. Possibly your IDE made this change automatically for some reason, but please be careful that you're not checking in unintended changes to the project configuration.